cmd/app.go

```diff
@@ -16,20 +16,32 @@ package cmd
 
 import (
 	"fmt"
-
+	"github.com/redhat-developer/ocdev/pkg/application"
 	"github.com/spf13/cobra"
+	"os"
 )
 
-// appCmd represents the app command
-var appCmd = &cobra.Command{
-	Use:   "app",
-	Short: "Create Application",
-	Long:  `-`,
+// applicationCmd represents the app command
+var applicationCmd = &cobra.Command{
+	Use:     "application",
+	Short:   "application",
+	Aliases: []string{"app"},
+}
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "create an application",
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Also this should error (show help) when no argument is given.

You can do it by adding `Args: cobra.ExactArgs(1)`
see: https://github.com/spf13/cobra/#positional-and-custom-arguments

Reply:
Thanks, TIL :)